internal/envoy/listener.go

```diff
@@ -99,6 +99,12 @@ func HTTPConnectionManager(routename string, accesslogger []*accesslog.AccessLog
 				}, {
 					Name: wellknown.Router,
 				}},
+				CommonHttpProtocolOptions: &envoy_api_v2_core.HttpProtocolOptions{
+					// Sets the idle timeout for HTTP connections to 60 seconds.
+					// This is chosen as a rough default to stop idle connections wasting resources,
+					// without stopping slow connections from being terminated too quickly.
+					IdleTimeout: protobuf.Duration(60 * time.Second),
```

Comment:
Do we document these defaults anywhere? Might be nice to show what's different from Envoy defaults.

Reply:
I opened https://github.com/projectcontour/contour/issues/1932 to track.